packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -749,6 +750,15 @@ class WebpackCLI implements IWebpackCLI {
         ],
         description: "Merge two or more configurations using 'webpack-merge'.",
       },
+      {
+        name: "disable-interpret",
+        configs: [
+          {
+            type: "boolean",
```

Comment:
Because with `type: "boolean"` we automatically add a negated option for it. Here, we don't wan't to create a `--no-disable-interpret` option.

```suggestion
            type: "enum",
            values: [true],
```



See [`fail-on-warnings` for reference](https://github.com/webpack/webpack-cli/blob/b5a2dc9f105a372229be7a8b1ae1ab8eb48baf65/packages/webpack-cli/src/webpack-cli.ts#L980-L987)

Reply:
@snitin315 

I see. I fixed the type.